### PR TITLE
VOTE: formation of documentation sub-team

### PR DESCRIPTION
**Pull request by croth1**
After getting some initial positive feedback, I would like to officially propose the sub-team formation of a doc team.

I would be happy to 
 - receive feedback on the sub-team description
 - find volunteers for the member list

I started by adding @scopatz, as he has worked with me on the restructuring in the last weeks. I hope this is ok.

@conda-forge/core - this is a sub-team formation vote and requires a 50% majority to pass, or 9/17 core members

xref: #729

### CODE REVIEW

File: src/orga/subteams.rst [hunk 1]
```diff
@@ -83,3 +83,41 @@ Members
...
+Maintain and improve the documentation; review, organize and help with documentation related issues. 
+
+Charter
+-------
+Dynamic
+
+Responsibility
+--------------
+
+A good documentation is an important corner stone of a successful community project.
```
Comment (jakirkham) [inline src/orga/subteams.rst]:
```suggestion
Good documentation is an important corner stone of a successful community project.
```

File: src/orga/subteams.rst [hunk 2]
```diff
@@ -83,3 +83,41 @@ Members
...
+An accurate, well organized and comprehensive documentation not only benefits users, but also frees the core team by decreasing support requests.
```
Comment (jakirkham) [inline src/orga/subteams.rst]:
```suggestion
Accurate, well organized and comprehensive documentation not only benefits users, but also frees the core team by decreasing support requests.
```

**Comment by croth1** [comment]
Thanks for your feedback @jakirkham

**Comment by scopatz** [comment]
BTW this needs 9 core members to approve

**Comment by croth1** [comment]
This is 9/18 🎉

**Comment by scopatz** [comment]
This still needs to be open for a week before it can be merged :)

**Comment by scopatz** [comment]
:tada:

**Pull request merged**